pulp_rpm/app/models.py

```diff
@@ -4,7 +4,7 @@
 import createrepo_c as cr
 
 from django.db import models
-from pulpcore.plugin.models import Content, Remote, Publisher
+from pulpcore.plugin.models import Content, Remote, Publication, Publisher
```

Comment:
any reason we are keeping rpm publisher model as well as its serializer?

Reply:
Nope. Good catch.